### PR TITLE
Post Install Support + bug fixes for Intersystems Cache installer

### PR DESCRIPTION
**Pull request by shabiel**

### CODE REVIEW

File: Common/pvPostInstall.sh [hunk 1]
```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+# NB NB NB: There are tabs in this code. They MUST be kept.
+# Needs a parameter: instance name ($1)
+instance=$1
+ccontrol start CACHE
+csession CACHE -U $instance <<END
+; Save pvPostInstall Routine
+W "Saving pvPostInstall...",!
+ZR  ZS pvPostInstall
```
Comment (ChristopherEdwards) [inline Common/pvPostInstall.sh]: why is this required here but not for KBANTCLN?

Comment (shabiel) [reply Common/pvPostInstall.sh]: You are right. I need to fix this.

File: Cache/install.sh [hunk 2]
```diff
@@ -120,11 +123,13 @@ fi
 cp $scriptdir/cache.cpf $basedir/cache.cpf-new
 perl -pi -e 's/foia/'$instance'/g' $basedir/cache.cpf-new
 perl -pi -e 's/FOIA/'${instance^^}'/g' $basedir/cache.cpf-new
+cp $basedir/cache.cpf-new $basedir/cache.cpf
```
Comment (ChristopherEdwards) [inline Cache/install.sh:126]: This is done on line 151, was there an issue?

Comment (shabiel) [reply Cache/install.sh:126]: It never happened for the install instance of the cache; so the post install didn't work.

Comment (ChristopherEdwards) [reply Cache/install.sh:126]: OK. I can see how that doesn't work :). carry on.
